flexget/plugins/input/discover.py

```diff
@@ -292,12 +289,22 @@ def interval_expired(self, config, task, entries):
         return result
 
     def on_task_input(self, task, config):
-        config.setdefault('release_estimations', {})
-        if not isinstance(config['release_estimations'], dict):
-            config['release_estimations'] = {'mode': config['release_estimations']}
+        config.setdefault('release_estimations', 'strict')
 
-        config['release_estimations'].setdefault('mode', 'strict')
-        config['release_estimations'].setdefault('optimistic', '0 days')
+        if not isinstance(config['release_estimations'], dict):
+            # 'optimistic' mode is being deprecated to add support for modifying the
+            # resultant release date through a consistent modifier used by all modes.
+            if 'optimistic' in config['release_estimations']:
+                # Identify configurations that have both the deprecated optimistic
+                # mode enabled and the new configuration option.
+                if config['release_date_modifier'] != '0 days':
+                    logger.critical("optimistic is deprecated - remove conflictig configuration.")
```

Comment:
Small typo here: conflictig > conflicting
Maybe give a little more context on "optimistic" telling it's about release estimations?